### ISSUE TITLE
Users are unable to log in

### ISSUE THREAD

**Issue by benoitchantre** [Automattic/maintenance-mode-wp#18]
When the maintenance mode is enabled, users cannot log in and update the website.

The login page should always be accessible.

When users try to log in without the required capability, they should be redirected to the maintenance page.

**Comment by mikeyarce** [comment]
@benoitchantre - I tested this out today and I found:

> When the maintenance mode is enabled, users cannot log in and update the website.
> The login page should always be accessible.

Users can log in and update the website when maintenance mode is enabled, and the login page is accessible.

Can you let me know if these two things are still issues for you?  If you still see this happening for you, can you tell me more about your setup and what else you might have going on your site?  Thanks so much!

> When users try to log in without the required capability, they should be redirected to the maintenance page.

Right now, if a user tries to log in without the required capability - they can still log in and are not redirected.  However, viewing the site will still direct them to the maintenance page.  I've added this as a separate issue so we can discuss it separately:
https://github.com/Automattic/maintenance-mode-wp/issues/30

**Comment by benoitchantre** [comment]
Thank you for your tests @mikeyarce.

> When the maintenance mode is enabled, users cannot log in and update the website.
The login page should always be accessible.

I did a new test on a development site and I wasn't able to reproduce this issue, even with OPcache enabled and configured with [production settings](https://www.scalingphpbook.com/blog/2014/02/14/best-zend-opcache-settings.html). Tested also when WordPress is installed in a subdirectory.

I think this issue can be closed.
